module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -0,0 +1,99 @@
+## Intoduction to database schemas
```

Comment:
in general titles should be: Cased Like This, but connecting works like `and, to`, etc are not capitalized